### PR TITLE
win: add disk read/write count to uv_getrusage

### PR DESCRIPTION
**Pull request by iWuzHere**
adding some improvements as discussed in https://github.com/libuv/libuv/issues/342

CI: https://ci.nodejs.org/view/libuv/job/libuv-test-commit/114/

### CODE REVIEW

File: src/win/util.c [hunk 1]
```diff
@@ -1102,6 +1103,12 @@ int uv_getrusage(uv_rusage_t *uv_rusage) {
     return uv_translate_sys_error(GetLastError());
   }
 
+  ret = GetProcessIoCounters(GetCurrentProcess(), &ioCounters);
+
```
Comment (saghul) [inline src/win/util.c]: extra newline

**Comment by saghul** [comment]
LGTM with a style nit. Can you add some "Refs: " metadata when you merge?

**Comment by iWuzHere** [comment]
Updated changeset, and will do!

**Comment by iWuzHere** [comment]
Landed https://github.com/libuv/libuv/commit/a7dfee3b02c70ca872c750fc60860d238623be69

**Pull request closed without merging**